.github/workflows/release.yml

```diff
@@ -2,12 +2,6 @@ name: Release and version
 
 on:
   workflow_dispatch:
-    inputs:
-      branch:
-        description: "Trigger a release workflow"
-        type: string
-        default: "develop"
-        required: true
```

Comment:
This was unnecessary might have been necessary in older versions of github actions